rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -123,6 +124,14 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
             take=1,
             method='spend',
         )
+        fee_key = str(HistoryEventType.SPEND) + '__' + str(HistoryEventSubType.FEE)
```

Comment:
There is already `get_tx_event_type_identifier(HistoryEventType.SPEND, HistoryEventSubType.FEE, CPT_ONEINCH_V1): TxEventSettings(  # noqa: E501` for fees. Shoudl it exist? It has `count_entire_amount_spend` as `False` though.

Reply:
Hm, I am not sure what value should `count_entire_amount_spend` have because I guess in different jurisdictions fees should be treated differently. I chose `True` here because gas by default is taxable and has `count_entire_amount_spend` set to `True`.